### PR TITLE
Fixed calculation for Ansible playbook service max TTL.

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
Updated method and test.
https://bugzilla.redhat.com/show_bug.cgi?id=1497983

@miq-bot add_label bug, fine/yes

### CODE REVIEW

**Comment by billfitzgerald0120** [comment]
@miq-bot assign @gmcculloug

**Comment by billfitzgerald0120** [comment]
@tinaafitz Please Review

**Comment by tinaafitz** [comment]
@billfitzgerald0120 Looks good.
@mkanoor Please review.

File: spec/content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/check_completed_spec.rb [hunk 1]
```diff
@@ -94,9 +94,9 @@
         Spec::Support::MiqAeMockObject.new('service' => svc_service, 'service_action' => 'Provision', 'ae_state_max_retries' => 100)
       end
 
-      context "600 ttl, 100 retries eq interval 6" do
-        let(:ae_retry_interval) { 6.minutes }
-        let(:ttl) { 600 }
+      context "650 ttl, 100 retries eq interval 6" do
```
Comment (gmcculloug) [inline spec/content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/check_completed_spec.rb]: Also need to update the "internal" value of the context.

File: spec/content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/check_completed_spec.rb [hunk 2]
```diff
@@ -117,9 +117,9 @@
         Spec::Support::MiqAeMockObject.new('service' => svc_service, 'service_action' => 'Retirement', 'ae_state_max_retries' => 100)
       end
 
-      context "600 ttl, 100 retries eq interval 6" do
-        let(:ae_retry_interval) { 6.minutes }
-        let(:ttl) { 600 }
+      context "650 ttl, 100 retries eq interval 6" do
```
Comment (gmcculloug) [inline spec/content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/check_completed_spec.rb]: Same here.

**Comment by billfitzgerald0120** [comment]
@gmcculloug Made changes as requested, please review

**Comment by simaishi** [comment]
Fine backport details:

```
$ git log -1
commit 0d4b9292182140c27c16a83a4ff7b7f0828b1180
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Thu Oct 5 13:06:49 2017 -0400

    Merge pull request #193 from billfitzgerald0120/timetolive_float
    
    Fixed calculation for Ansible playbook service max TTL.
    (cherry picked from commit ae2736d354c1b31624e2af9fbb967a419eabf9f8)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1501524
```

**Pull request merged**